src/pages/health-declaration/cat-adi-part2/health-declaration.cat-adi-part2.page.ts

```diff
@@ -28,11 +28,11 @@ import {
 // TODO - ADI Part 2: use correct reducer
 import { getCandidate } from '../../../modules/tests/journal-data/cat-be/candidate/candidate.cat-be.reducer';
 import { map, tap } from 'rxjs/operators';
-import {
-  getPassCertificateNumber,
-  isProvisionalLicenseProvided,
-} from '../../../modules/tests/pass-completion/pass-completion.selector';
-import { getPassCompletion } from '../../../modules/tests/pass-completion/pass-completion.reducer';
+// import {
+//   getPassCertificateNumber,
+//   isProvisionalLicenseProvided,
+// } from '../../../modules/tests/pass-completion/pass-completion.selector';
+// import { getPassCompletion } from '../../../modules/tests/pass-completion/pass-completion.reducer';
```

Comment:
should this commented out code be removed?

Reply:
will be removed after i rebase and your changes are in!